### ISSUE TITLE
Changing meteoriteMaximumSpawnHeight in generated config file seems to accomplish nothing.

### ISSUE THREAD

**Issue by Moleculor** [AppliedEnergistics/Applied-Energistics-2#4938]
**Describe the bug**
When launching an instance of Forge+Minecraft with AE2 installed, a file named `appliedenergistics2-common.toml` is generated automatically by the mod.

Within that file a section exists called `[worldGen]`, and within that section there is a line that reads `meteoriteMaximumSpawnHeight = 180` by default.

Changing this value does not reduce the maximum spawn height for meteorites.

**To Reproduce**
1. Install Minecraft 1.16, Forge 35.1.36, Applied Energistics 2 v8.1.0, and any other mods your heart desires.
2. Launch this collection. 
3. Close Minecraft once it has fully loaded.
4. Enter the directory containing the config files.
5. Change the line described above to much lower values (say, 30 or 40).
6. Relaunch this collection of Minecraft and mods.
7. Create a fresh, brand new world. 
8. Fly around in it with a Meteorite Compass, or use `/locate appliedenergistics2:meteorite` to locate a meteorite. 
9. Observe that said meteorite has likely spawned in a location far above the value entered in the config file.

**Expected behavior**
Meteors would only spawn in at or below the height set in the config. In prior versions of AE2, this meant that the only meteors that could or would be discovered might be underground ones.

**Additional context**
```
[worldGen]
	#Range: 0.0 ~ 1.0
	spawnChargedChance = 0.08
	meteoriteMaximumSpawnHeight = 30
```

![image](https://user-images.githubusercontent.com/3865976/104258312-e287ec00-5444-11eb-93cd-1ab955ddbe14.png)

**Environment**
```
-- System Details --
Details:
	Minecraft Version: 1.16.4
	Minecraft Version ID: 1.16.4
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 14.0.2, AdoptOpenJDK
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode, sharing), AdoptOpenJDK
	Memory: 3753666880 bytes (3579 MB) / 6442450944 bytes (6144 MB) up to 6442450944 bytes (6144 MB)
	CPUs: 8
	JVM Flags: 10 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xmx6144m -Xms6144m -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -XX:+UseG1GC -XX:+UnlockExperimentalVMOptions -XX:G1NewSizePercent=20 -XX:G1ReservePercent=20 -XX:MaxGCPauseMillis=50 -XX:G1HeapRegionSize=32M
	ModLauncher: 8.0.6+85+master.325de55
	ModLauncher launch target: fmlclient
	ModLauncher naming: srg
	ModLauncher services: 
		/mixin-0.8.2.jar mixin PLUGINSERVICE 
		/eventbus-3.0.5-service.jar eventbus PLUGINSERVICE 
		/forge-1.16.4-35.1.36.jar object_holder_definalize PLUGINSERVICE 
		/forge-1.16.4-35.1.36.jar runtime_enum_extender PLUGINSERVICE 
		/forge-1.16.4-35.1.36.jar capability_inject_definalize PLUGINSERVICE 
		/accesstransformers-2.2.0-shadowed.jar accesstransformer PLUGINSERVICE 
		/forge-1.16.4-35.1.36.jar runtimedistcleaner PLUGINSERVICE 
		/mixin-0.8.2.jar mixin TRANSFORMATIONSERVICE 
		/forge-1.16.4-35.1.36.jar fml TRANSFORMATIONSERVICE 
	FML: 35.1
	Forge: net.minecraftforge:35.1.36
	FML Language Providers: 
		javafml@35.1
		minecraft@1
	Mod List: 
		forge-1.16.4-35.1.36-client.jar                   |Minecraft                     |minecraft                     |1.16.4              |DONE      |NOSIGNATURE
		forge-1.16.4-35.1.36-universal.jar                |Forge                         |forge                         |35.1.36             |DONE      |22:af:21:d8:19:82:7f:93:94:fe:2b:ac:b7:e4:41:57:68:39:87:b1:a7:5c:c6:44:f9:25:74:21:14:f5:0d:90
		jeiintegration_1.16.4-6.1.1.11.jar                |JEI Integration               |jeiintegration                |6.1.1.11            |DONE      |NOSIGNATURE
		Xaeros_Minimap_20.30.1_Forge_1.16.4.jar           |Xaero's Minimap               |xaerominimap                  |20.30.1             |DONE      |NOSIGNATURE
		World Stripper-1.16.4-2.0.1.jar                   |World Stripper                |worldstripper                 |2.0.1               |DONE      |NOSIGNATURE
		XaerosWorldMap_1.11.6_Forge_1.16.4.jar            |Xaero's World Map             |xaeroworldmap                 |1.11.6              |DONE      |NOSIGNATURE
		appliedenergistics2-8.2.0-beta.2.jar              |Applied Energistics 2         |appliedenergistics2           |8.2.0-beta.2        |DONE      |95:58:cc:83:9d:a8:fa:4f:e9:f3:54:90:66:61:c8:ae:9c:08:88:11:52:52:df:2d:28:5f:05:d8:28:57:0f:98
		jei-1.16.4-7.6.1.65.jar                           |Just Enough Items             |jei                           |7.6.1.65            |DONE      |NOSIGNATURE
	Crash Report UUID: 24815fa7-21d7-45f1-94c3-fade09ec478b
	Launched Version: 1.16.4
	Backend library: LWJGL version 3.2.2 build 10
	Backend API: GeForce RTX 2070/PCIe/SSE2 GL version 4.6.0 NVIDIA 461.09, NVIDIA Corporation
	GL Caps: Using framebuffer using OpenGL 3.0
	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'forge'
	Type: Client (map_client.txt)
	Graphics mode: fancy
	Resource Packs: 
	Current Language: English (US)
	CPU: 8x Intel(R) Core(TM) i7-4790K CPU @ 4.00GHz
```

**Comment by shartte** [comment]
This config option has already been removed in newer versions.

**Comment by Moleculor** [comment]
@shartte 
> This config option has already been removed in newer versions.

Sorry, in my initial post I assumed I had installed the release version when in fact I had installed the 8.2.0 beta 2. 

So are you referring to unreleased versions?

Also... how do I prevent meteorites from spawning on the surface of the world in upcoming versions? There was actually a recent conversation on [/r/feedthebeast](https://www.reddit.com/r/feedthebeast/comments/koxpcg/traveled_thousands_of_blocks_to_find_a_woodland/) about how meteors were undesired by many, so I know I'm not the only one looking to do so.
